layersvt/VkLayer_device_simulation.json.in

```diff
@@ -65,6 +65,54 @@
                             "value": "${VULKAN_CONTENT}/VK_LAYER_LUNARG_device_simulation/iOS_gpu_family_5_portability.json"
                         }
                     ]
+                },
+                {
+                    "label": "Vulkan 1.2 minimum requirements",
```

Comment:
I think we are missing preset for the portability flavors of the presets.

Should portability subset enabled for these presets?

Reply:
I was thinking it would be good for the presets to also include the portability files we already ship, like the macOS and iOS portability files, in the filenames with these.

To use multiple config files in Devsim, you add file paths to the `VK_DEVSIM_FILENAMES` environment variable separated by the `;` on Windows or the `:` on Linux and macOS. ​

Is there a substitution variable like the `${VULKAN_CONTANT}` variable for system specific delimiters?

Is there some other method in VkConfig to select multiple files or multiple presets?

Reply:
Alright, I'll save those for a future PR.

To answer your question about multiple config files setting the same limit with different values, the limit is set to the value in the last config file in the filenames list.